hypothesis-python/src/hypothesis/core.py

```diff
@@ -106,10 +106,18 @@
     MappedSearchStrategy,
     SearchStrategy,
 )
-from hypothesis.utils.conventions import InferType, infer
+from hypothesis.utils.conventions import infer
 from hypothesis.vendor.pretty import RepresentationPrinter
 from hypothesis.version import __version__
 
+if sys.version_info >= (3, 10):  # pragma: no cover
+    from types import EllipsisType
+
+    InferType = EllipsisType
```

Comment:
```suggestion
    from types import EllipsisType as EllipsisType
```

Reply:
Are you saying:

```python
if sys.version_info >= (3, 10):  # pragma: no cover
    from types import EllipsisType
else:
    EllipsisType = type(Ellipsis)
```

(which, in retrospect, makes sense). But why `from types import EllipsisType as EllipsisType`?

Reply:
🤦‍♂️ no, typo fixed, I meant *`as InferType`*